.github/workflows/security-scan.yml

```
@@ -0,0 +1,165 @@
+      uses: dependency-check/Dependency-Check_Action@main
+      with:
+        project: 'OpenLegislation'
+        path: '.'
+        format: 'ALL'
+        args: >
+          --enableRetired
+          --enableExperimental
+          --nvdValidForHours 24
+
+    - name: Upload dependency check results
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.8)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)